qiskit/dagcircuit/dagcircuit.py

```diff
@@ -889,10 +889,11 @@ def __eq__(self, other):
         # Try to convert to float, but in case of unbound ParameterExpressions
         # a TypeError will be raise, fallback to normal equality in those
         # cases
+
         try:
             self_phase = float(self.global_phase)
             other_phase = float(other.global_phase)
-            if not np.isclose(self_phase, other_phase):
+            if abs((self_phase - other_phase + np.pi) % (2 * np.pi) - np.pi) > 1.E-10:  # XXX: atol?
```

Comment:
TODO takes it over 100chars for the line :) All tools I know about treat XXX, FIXME and TODO the same so I have a habit of using XXX

Since qiskit standardized on TODO I'll change it